pkg/tsdb/loki/instrumentation/instrumentation.go

```diff
@@ -18,12 +19,8 @@ var (
 	}, []string{"status", "endpoint"})
 )
 
-const (
-	EndpointQueryData = "queryData"
-)
-
 func UpdatePluginParsingResponseDurationSeconds(ctx context.Context, duration time.Duration, status string) {
-	histogram := pluginParsingResponseDurationSeconds.WithLabelValues(status, EndpointQueryData)
+	histogram := pluginParsingResponseDurationSeconds.WithLabelValues(status, string(backend.EndpointQueryData))
```

Comment:
I assume this will not change the value of `backend.EndpointQueryData`? So it stays the string `queryData`.

Reply:
Yes correct